tags/resources/toc-supporting-guides/tech-papers.md

```diff
@@ -0,0 +1,188 @@
+
+Adjudicators may be drawn from contributors to the paper, as they often have additional context that is assummed in the paper, but not declared until a commenter points it out. It is recommended that adjudicators be individuals most heavily involved in the operations of paper development, not necessarily tied to a specific topic or section (although perhaps they wrote many of them). They should be able to reason and apply the comment in the context of the section, as well as across the body of the paper, while still taking a step back to check assumptions in content.
+
+#### Leadership Review and Approval
+
+Once public comments are adjudicated, to ensure appropriate oversight on content, two additional reviews are required before the paper may be published.
+
+##### TAG Sponsor and TOC Liaison review and approval
+
+The TAG Sponsor must review and provide their approval on the finalized draft. Their review and approval signifies the paper as ready for the TOC Liaisons to review and approve.
+
+The TAG Sponsor will work with the TAG Co-Chairs to ensure the TOC Liaisons are aware of their pending review and approval.
+
+All approvals are to be reflected in the metadata heading of the document, the following example may be used:
+
+**Version**: 2.0 **Created**: 12 Jan 2022 **Status**: WIP | In Review | **Approved**
+
+**Last Reviewed**: 16 May 2022, **PDF Published**: 17 May 2022 **Release Version**: 2.0
+
+**TAG Sponsor Approver** [X] $GITHUBHANDLE
+**TOC Liaison Approvers** [X] $GITHUBHANDLE [X] $GITHUBHANDLE
+
+**Version 2 (May 2022)**
+
+* **Contributors**: $NAME or $GITHUBHANDLE
+* **Reviewers**: $NAME or $GITHUBHANDLE
+
+#### CNCF Publishing Engagement
+
+In order to initiate publishing of the paper, a TAG co-chair must file a Service Desk ticket using the [CNCF Service Desk](https://cncfservicedesk.atlassian.net/servicedesk/customer/portal/1/user/login?destination=portal%2F1). If a TAG co-chair does not have a service desk account, reach out to CNCF staff to have the account set up. In the service desk ticket, you'll need to let staff know what you're looking for, who is the primary (TAG Leadership sponsor) and secondary contact, and where the materials may be found for publication. 
+
+The TAG Leadership sponsor works with CNCF for final edits, PDF conversion, and graphics inclusion. Review drafts before the final version is added to the repo to ensure accuracy and clarity. Note: papers do not need to be PDFs, however the markdown must be present in the TAG repo.
+
+#### Addition to the Repo
```

Comment:
Should we also have a section on updating the paper?  What would be guidelines in terms of either updating an existing paper or starting a new one?